MODULE.bazel

```diff
@@ -16,7 +16,7 @@
 
 module(
     name = "rules_pkl",
-    version = "0.13.1",
+    version = "0.14.0",
     bazel_compatibility = [">=7.0.0"],
```

Comment:
Given [this](https://github.com/apple/rules_pkl/blob/76d974b078d4bda82bc8bced6b051d8768caac71/MODULE.bazel#L107), should we update this to `>=7.6.1` ?